sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UnusedPrivateMember.cs

```diff
@@ -85,7 +85,7 @@ protected override void Initialize(SonarAnalysisContext context)
                             // Collect symbols of private members that could potentially be removed
                             var removableSymbolsCollector = new CSharpRemovableSymbolWalker(c.Compilation.GetSemanticModel);
 
-                            if (!VisitDeclaringReferences(namedType, removableSymbolsCollector, c.Compilation))
+                            if (!VisitDeclaringReferences(namedType, removableSymbolsCollector, c.Compilation, includeGeneratedFile: false))

```

Comment:
We don't need to collect the members declared in generated files as we don't want to report about them...